app/assets/css/syntax.css

```diff
@@ -1,227 +1,357 @@
-pre,
-code {
-    font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
-    max-width: 738px; /* this is inside a table-cell */
-}
-pre {
-    padding: 10px 16px 10px 7px;
-    line-height: 1.45;
-    border-radius: 3px;
-    background: url(https://s.yimg.com/os/acss/images/pre-background.177ebfe5.gif);
-}
-
 /* https://github.com/jwarby/jekyll-pygments-themes/blob/master/github.css */
 .highlight {
     margin: 0 0 1em;
 }
+
 .highlight .hll {
     background-color: #ffffcc;
```

Comment:
var?

Reply:
Only used once, so left hardcoded. Did that for many of the syntax colors.